### PR TITLE
More content updates

### PR DESCRIPTION
**Pull request by michaelclement**
## Summary

This PR introduces the following content updates:

**Signer information form section**
- Changes 2nd radio option text
- Changes "Your information" to "Your name" on third party page
- Removes subtitle under H3 on /certification/name page
- Removes "If you don't have an address" on /certification/address page
- Updates certification relationship subtitle

**Sponsor information form section**
- Updates subtitle on sponsor name-dob page
- Removes subtitle from sponsor ssn page and adds hint to VA file number
- Requires an answer to sponsor death happening during active service
- Changes subtitle text on sponsor address page
- Updates subtitle on sponsor phone page

**Applicant information form section**
- Removes subtitle text and adds hint to VA file number on ssn page
- Updates the pre-address screener page with new wording and functionality

...

- Team: IVC CHAMPVA
- Flipper: NA

## Testing done

- Manual testing
- Verified existing tests pass

## Screenshots

_Note: This field is mandatory for UI changes (non-component work should NOT have screenshots)._

|         | Updated address selector |
| ------- | ----- |
| Desktop |Screenshot 2024-03-25 at 16 15 50|

## What areas of the site does it impact?

*(Describe what parts of the site are impacted **if** code touched other areas)*

### Quality Assurance & Testing

- [X] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [X] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [X] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [X] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [X] Browser console contains no warnings or errors.
- [X] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:

- [ ] The vets-website header does not contain any web-components
- [ ] I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#local-dev) to test the injected header scenario
- [ ] I reached out in the `#sitewide-public-websites` Slack channel for questions

## Requested Feedback

NA

### CODE REVIEW

File: src/applications/ivc-champva/10-10D/components/CustomRelationshipPattern.jsx [hunk 1]
```diff
@@ -1,118 +1,4 @@
-import {
```
Comment (michaelclement) [inline src/applications/ivc-champva/10-10D/components/CustomRelationshipPattern.jsx:1]: Unused, so removed whole component.

File: src/applications/ivc-champva/10-10D/components/CustomSsnPattern.jsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        }
+
+        return {
+          ..._schema,
+          required,
+        };
+      },
+    },
+  };
+};
```
Comment (michaelclement) [inline src/applications/ivc-champva/10-10D/components/CustomSsnPattern.jsx:34]: Need to have the behavior of the stock ssn/vafilenumber, but with an extra hint over va file number.

File: src/applications/ivc-champva/10-10D/pages/ApplicantAddressPage.jsx [hunk 3]
```diff
@@ -1,11 +1,9 @@
 import React, { useState, useEffect } from 'react';
```
Comment (michaelclement) [inline src/applications/ivc-champva/10-10D/pages/ApplicantAddressPage.jsx:1]: Removed radio button and added certifier address + sponsor address to list of values we can choose from.

File: src/applications/ivc-champva/10-10D/pages/Sponsor/sponsorInfoConfig.js [hunk 4]
```diff
@@ -9,14 +9,30 @@ import {
...
+
 export const sponsorNameDobConfig = {
   uiSchema: {
     ...titleUI(
       ({ formData }) => `${sponsorWording(formData)} name and date of birth`,
-      ({ formData }) =>
-        formData?.certifierRole === 'sponsor'
-          ? 'Please provide your information. We use this information to identify eligibility.'
-          : `Enter the information for your sponsor (this is the Veteran or service member that you’re connected to). We’ll use the sponsor’s information to confirm your eligibility for CHAMPVA benefits.`,
+      ({ formData }) => descriptionText(formData),
```
Comment (michaelclement) [inline src/applications/ivc-champva/10-10D/pages/Sponsor/sponsorInfoConfig.js:35]: Added additional customized wording and pulled some logic up out of the ternary

File: src/applications/ivc-champva/10-10D/tests/components/CustomRelationshipPattern.unit.spec.js [hunk 5]
```diff
@@ -1,23 +0,0 @@
...
-    });
-    expect(relationship.relationshipToVeteran['ui:title']).to.exist;
-  });
-  it('should allow past-tense wording', () => {
-    const relationship = relationshipToVeteranUI({ tense: 'past' });
-    expect(relationship.relationshipToVeteran?.['ui:title']).to.contain(
-      'What’s your relationship',
-    );
-  });
-});
```
Comment (michaelclement) [inline src/applications/ivc-champva/10-10D/tests/components/CustomRelationshipPattern.unit.spec.js:23]: Tests no longer needed now that the component is gone.

**Review by balexandr — APPROVED** [review]
Lgtm

**Review by cloudmagic80 — APPROVED** [review]

**Review by jamigibbs — APPROVED** [review]

**Pull request merged**